force-app/main/default/classes/IPayment.cls

```diff
@@ -0,0 +1,7 @@
+public Interface IPayment {
```

Comment:
"Interface" should be "interface", won't matter when it compiles but helpful fo readability. You'll notice it turns the color red.

Reply:
Same comment thanks @acann22!